mp4parse/src/lib.rs

```diff
@@ -3081,7 +3087,7 @@ fn read_mdia<T: Read>(f: &mut BMFFBox<T>, track: &mut Track) -> Result<()> {
                 debug!("{:?}", mdhd);
             }
             BoxType::HandlerBox => {
-                let hdlr = read_hdlr(&mut b)?;
+                let hdlr = read_hdlr(&mut b, ParseStrictness::Permissive)?;
```

Comment:
When parsing non-AVIF I left the behavior unchanged

Reply:
Thanks, looks good.  We used to read the component name field, but saw [various invalid forms](https://github.com/mozilla/mp4parse-rust/commit/012fd663126e8812aa2d0355b0823cf1e082df2c?branch=012fd663126e8812aa2d0355b0823cf1e082df2c&diff=unified#diff-f34d58d35412a7aa40d5fddc3be5b95878557f09b0618633c2654e1bf21ebfceL2001) in the wild and eventually dropped the code since we never used the field.